internal/services/network/subnet_data_source.go

```diff
@@ -64,18 +65,42 @@ func dataSourceSubnet() *pluginsdk.Resource {
 					Type: pluginsdk.TypeString,
 				},
 			},
+		},
+	}
 
-			"enforce_private_link_endpoint_network_policies": {
-				Type:     pluginsdk.TypeBool,
-				Computed: true,
-			},
+	if features.FourPointOhBeta() {
+		resource.Schema["private_endpoint_network_policies_enabled"] = &pluginsdk.Schema{
+			Type:     pluginsdk.TypeBool,
+			Computed: true,
+		}
 
-			"enforce_private_link_service_network_policies": {
-				Type:     pluginsdk.TypeBool,
-				Computed: true,
-			},
-		},
+		resource.Schema["private_link_service_network_policies_enabled"] = &pluginsdk.Schema{
+			Type:     pluginsdk.TypeBool,
+			Computed: true,
+		}
+	} else {
```

Comment:
(same here)

Reply:
Totally makes sense for the data source... fixed.